frontend/src/components/Table/TableHealthChip/index.tsx

```diff
@@ -1,16 +1,21 @@
-import { Chip } from "@mui/material";
-import { Irgb } from "types";
+import { Chip, Tooltip } from "@mui/material";
+import { ITableHealthChip } from "types";
 
-export const TableHealthChip = ({ rgb }: Irgb) => {
-  const { red, green, blue, health_name } = rgb;
+export const TableHealthChip = ({ color, caption }: ITableHealthChip) => {
+  const { red, green, blue } = "string" === typeof color ? { red: 0, green: 0, blue: 0 } : color;
```

Comment:
So I assume when `color` comes from the backend it's in the form `{red: <number>, green: <number>, blue: <number>}` but when it comes from the `legendValues` array, the `color` value is a hex string? Could we define  `color` in `legendValues` the same way as the backend returns it so we don't have to do this type checking here?